### ISSUE TITLE
Unhandled promise rejection warning when scheduling jobs

### ISSUE THREAD

**Issue by reefdog** [TechAndCheck/tech-and-check-alerts#150]
When running `yarn queue:jobs:schedule`, the console dumps out the following before starting to log crawl/scrape progress:

```
Justins-Retina-MBP:tech-and-check-alerts justin$ yarn queue:jobs:schedule
yarn run v1.17.3
$ babel-node -- src/scripts/scheduleJobs
(node:60177) UnhandledPromiseRejectionWarning: Error: Invalid characters, got value: undefinedundefinedundefined
    at Function._parseField (/Users/justin/Projects/duke/tech-and-check-alerts/node_modules/cron-parser/lib/expression.js:186:11)
    at parse (/Users/justin/Projects/duke/tech-and-check-alerts/node_modules/cron-parser/lib/expression.js:774:36)
    at Function.parse (/Users/justin/Projects/duke/tech-and-check-alerts/node_modules/cron-parser/lib/expression.js:835:10)
    at Function.parseExpression (/Users/justin/Projects/duke/tech-and-check-alerts/node_modules/cron-parser/lib/parser.js:41:25)
    at getNextMillis (/Users/justin/Projects/duke/tech-and-check-alerts/node_modules/bull/lib/repeatable.js:202:29)
    at Queue.nextRepeatableJob (/Users/justin/Projects/duke/tech-and-check-alerts/node_modules/bull/lib/repeatable.js:33:24)
    at /Users/justin/Projects/duke/tech-and-check-alerts/node_modules/bull/lib/queue.js:688:19
    at async Promise.all (index 0)
(node:60177) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:60177) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by reefdog** [comment]
Actually, this is due to the `Schedules.None` typo in these three queues:

https://github.com/TechAndCheck/tech-and-check-alerts/blob/a7b63e1fe2e58049e95bc4acd6d50e0ca13a767e/src/server/queues/cnnTranscriptListCrawlerQueue/CnnTranscriptListCrawlerJobScheduler.js#L8
https://github.com/TechAndCheck/tech-and-check-alerts/blob/a7b63e1fe2e58049e95bc4acd6d50e0ca13a767e/src/server/queues/cnnTranscriptListCrawlerQueue/CnnTranscriptListCrawlerJobScheduler.js#L8
https://github.com/TechAndCheck/tech-and-check-alerts/blob/e6d944cb63850fa1bc05347cdb2aad5feaa66964/src/server/queues/claimBusterClaimDetectorQueue/ClaimBusterClaimDetectorJobScheduler.js#L8

This should be `Schedules.NONE` (and, actually, should be `SCHEDULES.NONE` but that's a different issue).

**Comment by reefdog** [comment]
Fixed the `NONE` typos in #157, but that just reduced our three `undefined`s down to two. Leaving this open.

**Comment by reefdog** [comment]
This was a compound issue: we aren't correctly ignoring jobs with no schedule, and this failure wasn't handled by our job-scheduling tasks. Splitting the former problem off to #172 and focusing this issue on the latter.
